src/transforms/groupby.js

```diff
@@ -35,6 +35,35 @@ exports.attributes = {
             'with `x` [1, 3] and one trace with `x` [2, 4].'
         ].join(' ')
     },
+    nameformat: {
```

Comment:
... and I would make user spell out `%{name}` and `%{group}` as opposed to e.g. `%t` and `%g`